fe/fe-core/src/test/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctionsTest.java

```diff
@@ -551,6 +551,46 @@ public void curDate() {
         assertEquals(now, ScalarOperatorFunctions.curDate().getDate());
     }
 
+
+    @Test
+    public void nextDay() {
```

Comment:
How about writing tests with sql-test which is end to end and easy to write

Reply:
Yes, I can rewrite the tests.
For `sql-test`, do you mean the `SQL-tester` framework in `starrocks/test/` path?

Reply:
Yes

Reply:
The `SQL-tester` framework is really convenient! I have added the tests and passed.